src/api/app/jobs/send_event_emails.rb

```diff
@@ -7,7 +7,12 @@ class SendEventEmails
   def perform
     Event::Base.where(mails_sent: false).order(created_at: :asc).limit(1000).each do |event|
       subscribers = event.subscribers
-      next if subscribers.empty?
+
+      if subscribers.empty?
+        event.update_attributes(mails_sent: true)
+        next
+      end
```

Comment:
Just a thought, but would it be possible to query all events with mails_sent false and subscribers empty (NULL?) and update them at once?

And then run the mails_sent false but subscribers not empty in a separate query. I don't have any numbers to back this up, but I could imagine this (the first query) would be faster.